spec/controllers/forms_controller_spec.rb

```diff
@@ -94,6 +107,19 @@
         end
       end
     end
+    context 'current user is nil' do
```

Comment:
Auto-filling one field with an attribute of another field. Kind of an edge case, but since we're writing the word bananas three times anyways, seems like we might as well make them different words.